plottable.d.ts

```diff
@@ -204,17 +208,14 @@ declare module Plottable {
              */
             has(key: K): boolean;
             /**
-             * Return an array of the values in the Map
-             *
-             * @return {V[]} The values in the store
-             */
-            values(): V[];
-            /**
-             * Return an array of keys in the Map.
+             * The forEach method executes the provided callback once for each key of the map which
+             * actually exist. It is not invoked for keys which have been deleted.
+             * However, it is executed for values which are present but have the value undefined.
```

Comment:
I guess my concern is mainly synchronization issues.  If MDN decides to change their documentation, we should change our documentation accordingly which... means we have to update consistently.  I think the better solution is say that this is a shim, provide our own description and then link.